frontend/src/app/teams/components/layout/teams-layout.component.scss

```diff
@@ -1,3 +1,63 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
 .container {
 	height: calc(100vh - 170px);
 }
+
+h1{
```

Comment:
Use specific class names to adjust the styles. since there could be a different h1 and h... 
Specify classes like teams__error-title